server/opts.go

```diff
@@ -57,20 +57,21 @@ func NoErrOnUnknownFields(noError bool) {
 // NOTE: This structure is no longer used for monitoring endpoints
 // and json tags are deprecated and may be removed in the future.
 type ClusterOpts struct {
-	Name           string            `json:"-"`
-	Host           string            `json:"addr,omitempty"`
-	Port           int               `json:"cluster_port,omitempty"`
-	Username       string            `json:"-"`
-	Password       string            `json:"-"`
-	AuthTimeout    float64           `json:"auth_timeout,omitempty"`
-	Permissions    *RoutePermissions `json:"-"`
-	TLSTimeout     float64           `json:"-"`
-	TLSConfig      *tls.Config       `json:"-"`
-	TLSMap         bool              `json:"-"`
-	ListenStr      string            `json:"-"`
-	Advertise      string            `json:"-"`
-	NoAdvertise    bool              `json:"-"`
-	ConnectRetries int               `json:"-"`
+	Name             string            `json:"-"`
+	Host             string            `json:"addr,omitempty"`
+	Port             int               `json:"cluster_port,omitempty"`
+	Username         string            `json:"-"`
+	Password         string            `json:"-"`
+	AuthTimeout      float64           `json:"auth_timeout,omitempty"`
+	Permissions      *RoutePermissions `json:"-"`
+	TLSTimeout       float64           `json:"-"`
+	TLSConfig        *tls.Config       `json:"-"`
+	TLSMap           bool              `json:"-"`
+	TLSImplicitAllow bool              `json:"-"`
```

Comment:
To be honest, it is not clear to me what this option does. Default will be false, so can you describe to me what setting this boolean to true will mean?

Reply:
the doc will say:
verify_and_implicit_allow| Only settable in a non client context where verify=true is the default (gateways/cluster). The incoming connections certificate's `X509v3 Subject Alternative Name` DNS entries will be matched against all urls in the configuration context that contains this tls map. If a match is found the connection is accepted, otherwise rejected. Meaning for gateways we will match all DNS entries in the certificate against all gateway urls. For cluster we will match against all route urls. As a consequence of this, dynamic cluster growth may require config changes in other cluster where this flag is true. DNS name checking is performed according to https://tools.ietf.org/html/rfc6125#section-6.4.1 Only the full wildcard `*` is supported for the left most label.

Reply:
clearer?

Reply:
The explanation seem clear to me, but I am not sure about the option name. Maybe @philpennock and @davidkemper can comment on that?